<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,6,1">
   <item>
    <widget class="QWidget" name="widget_FilePath" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_FilePath">
        <property name="text">
         <string>Upload File</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_FilePath</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_FilePath"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_FilePath">
        <property name="text">
         <string>Select File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_RequiredParameter">
     <property name="title">
      <string>Update File parameter</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_FileID">
        <property name="text">
         <string>FileID</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_FileID</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_FileID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_uploadType">
        <property name="text">
         <string>uploadType</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_uploadType</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_uploadType">
        <item>
         <property name="text">
          <string>media(Simple upload)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>multipart(Multipart upload)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>resumable(Resumable upload)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_OptionalParameter">
     <property name="title">
      <string>Optional parameter</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_addParents">
        <property name="text">
         <string>addParents</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_addParents</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_addParents"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_enforceSingleParent">
        <property name="text">
         <string>enforceSingleParent</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_enforceSingleParent</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_enforceSingleParent">
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_keepRevisionForever">
        <property name="text">
         <string>keepRevisionForever</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_keepRevisionForever</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_keepRevisionForever">
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_ocrLanguage">
        <property name="text">
         <string>ocrLanguage</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_ocrLanguage</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_ocrLanguage"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_removeParents">
        <property name="text">
         <string>removeParents</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_removeParents</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_removeParents"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_useContentAsIndexableText">
        <property name="text">
         <string>useContentAsIndexableText</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_useContentAsIndexableText</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="comboBox_useContentAsIndexableText">
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_supportsAllDrives">
        <property name="text">
         <string>supportsAllDrives</string>
        </property>
        <property name="buddy">
         <cstring>comboBox_supportsAllDrives</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBox_supportsAllDrives">
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UpdateDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
